frontend/src/app/app.module.ts

```diff
@@ -46,6 +48,10 @@ const appRoutes: Routes = [
     path: 'regulations',
     component: RegulationsComponent
   },
+  {
+    path: 'offers/:offerSlug/:offerId',
```

Comment:
I'm not sure what was the reason, but I think @radoslawroszkowiak wanted to have it this way?

Reply:
Yeah, we have the same in backend already. Let's leave it like it is.

Reply:
I've never wanted it that way ;)
The thing is THEY agreed to have it that way before. Don't ask me why :)

Reply:
@jacol12345 since the beggining the url was designed to show slug and after that id.

The reason for that is that id is unique and slug can be changed (after changing offer title) and we want show more human readable url.

Reply:
Slug should be unique and the old one should probably redirect if ever changed. Well, that's the separate discussion.

Reply:
We do redirect when slug changes in old view, but we don't require slug to be unique (that's the reason, we add id at the end).

Reply:
(but redirect of slug here is out of scope, will be done in separate issue)